tools/unused_modules.py

```diff
@@ -0,0 +1,75 @@
+import os
+import re
+
+rolesPath = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', 'roles'))
```

Comment:
The modules folder and roles folder have variables in cmake, and therefore the b script so grab them from there